### ISSUE TITLE
As a developer, I should be able to try out MyUSA on the dev server

### ISSUE THREAD

**Issue by ultrasaurus** [18F/midas#511]
Somehow we lost that config in recent "improvements" of automated deploy.  Need to set it up again.

**Comment by ultrasaurus** [comment]
Took a quick look at this with @cpapazian and found a few things
1) added @dhcole to the midas-dev MyUSA developer account -- you should be able to log in with your gsa.gov email and adjust settings on the app if needed
2) the params passed to MyUSA are incorrect
```
https://alpha.my.usa.gov/oauth/authorize?response_type=code&redirect_uri=http%3A%2F%2Flocalhost%2Fapi%2Fauth%2Fcallback%2Fmyusa&scope=profile.email%20profile.first_name%20profile.last_name&client_id=CLIENT_ID
```
2) CLIENT_ID is correct in local.js, but the app is reading from settings/auth.js
3) redirect url is also wrong, likely same root cause

Since the config was recently moved around, I figured it might make sense for @dhcole to take a look in the morning

**Comment by dhcole** [comment]
Yea, I've got this working locally. Not sure why sails is not merging in the local.js config, but that seems like another issue. Works fine with ENV vars.

**Comment by ultrasaurus** [comment]
getting closer, but still not working, I get this error
```The redirect uri included is not valid.```

This is the URL:
```https://alpha.my.usa.gov/oauth/authorize?response_type=code&redirect_uri=https%3A%2F%2Fmidas-dev.18f.us%2Fapi%2Fauth%2Fcallback%2Fmyusa&scope=profile.email%20profile.first_name%20profile.last_name&client_id=e8a52d3cfb9602cc6ff5b1f682a42cc4896b2859ec893f4b4d16aea23e97f035```

**Comment by cpapazian** [comment]
parsing the url escaping, it looks like the redirect in the querystring is https://midas-dev.18f.us/api/auth/callback/myusa ... the record in our system is http://midas-staging.18f.us/api/auth/callback/myusa

**Comment by dhcole** [comment]
@ultrasaurus do you have access to change that on https://alpha.my.usa.gov/authorizations? I don't have access to that application.

**Comment by cpapazian** [comment]
i can make the change. i assume midas-dev is correct?

**Comment by ultrasaurus** [comment]
@cpapazian this is the record I was looking at which had midas-dev.18f.us : https://alpha.my.usa.gov/applications/16/edit

**Comment by dhcole** [comment]
`https://midas-dev.18f.us/api/auth/callback/myusa`

Perhaps the token we're using is associated with a different application? @ultrasaurus do you have the tokens for Midas dev? Is the client id the same as the request? `e8a52d3cfb9602cc6ff5b1f682a42cc4896b2859ec893f4b4d16aea23e97f035`

**Comment by ultrasaurus** [comment]
@dhcole that's what I have saved for midas-dev, according to my local notes

**Comment by cpapazian** [comment]
there aren't any other client apps in the alpha. would have a different error if the token was bad.

**Comment by dhcole** [comment]
Closed by #608
